stable/velero/ci/test-values.yaml

```diff
@@ -0,0 +1,169 @@
+    config: {}
+    #  region:
+    #  s3ForcePathStyle:
+    #  s3Url:
+    #  kmsKeyId:
+    #  resourceGroup:
+    #  storageAccount:
+    #  publicUrl:
+
+  # Parameters for the `default` VolumeSnapshotLocation. See
+  # https://heptio.github.io/velero/v0.11.0/api-types/volumesnapshotlocation.html
+  volumeSnapshotLocation:
+    # Cloud provider where volume snapshots are being taken. Usually
+    # should match `configuration.provider`. Required.,
+    name:
+    # Additional provider-specific configuration. See link above
+    # for details of required/optional fields for your provider.
+    config: {}
+  #    region:
+  #    apitimeout:
+
+  # These are server-level settings passed as CLI flags to the `velero server` command. Velero
+  # uses default values if they're not passed in, so they only need to be explicitly specified
+  # here if using a non-default value. The `velero server` default values are shown in the
+  # comments below.
+  # --------------------
+  # `velero server` default: 1m
+  backupSyncPeriod:
+  # `velero server` default: 1h
+  resticTimeout:
+  # `velero server` default: namespaces,persistentvolumes,persistentvolumeclaims,secrets,configmaps,serviceaccounts,limitranges,pods
+  restoreResourcePriorities:
+  # `velero server` default: false
+  restoreOnlyMode:
+
+  # additional key/value pairs to be used as environment variables such as "AWS_CLUSTER_NAME: 'yourcluster.domain.tld'"
+  extraEnvVars: {}
+
+##
+## End of backup/snapshot location settings.
+##
+
+
+##
+## Settings for additional Velero resources.
+##
+
+# Whether to create the Velero cluster role binding.
+rbac:
+  create: true
+
+# Information about the Kubernetes service account Velero uses.
+serviceAccount:
+  server:
+    create: true
+    name: velero
+
+# Info about the secret to be used by the Velero deployment, which
+# should contain credentials for the cloud provider IAM account you've
+# set up for Velero.
+credentials:
+  # Whether a secret should be used as the source of IAM account
+  # credentials. Set to false if, for example, using kube2iam or
+  # kiam to provide IAM credentials for the Velero pod.
+  useSecret: true
+  # Name of a pre-existing secret (if any) in the Velero namespace
+  # that should be used to get IAM account credentials. Optional.
+  existingSecret:
+  # Data to be stored in the Velero secret, if `useSecret` is
+  # true and `existingSecret` is empty. This should be the contents
+  # of your IAM credentials file.
+  secretContents: {}
+
+# Whether to deploy the restic daemonset.
+deployRestic: false
+
+# Backup schedules to create.
+# Eg:
+# schedules:
+#   mybackup:
+#     schedule: "0 0 * * *"
+#     template:
+#       ttl: "240h"
+#       includedNamespaces:
+#        - foo
+schedules: {}
+
+# Whether or not to clean up CustomResourceDefintions when deleting a release.
+# This is useful in case there's multiple releases deployed and deleting the CRDs when deleting one would render the other broken
+cleanUpCRDs: true
```

Comment:
It's usually best to keep ci values files as minimal as possible in order to prevent config drift. If this is the only value that needs to be different from `values.yaml` then let's omit everything else.

Reply:
Done.